tar_scm.py

```diff
@@ -885,6 +904,7 @@ def parse_args():
         args.submodules = True
     else:
         args.submodules = False
+    args.sslverify = True if args.sslverify == 'enable' else False
```

Comment:
might be safer to reverse this logic, to only use False when there is an explicit "disable"